src/extension.ts

```diff
@@ -17,12 +17,15 @@ import { StatusMessages } from './statusMessages';
 import { caStatusBarProvider } from './caStatusBarProvider';
 import { CANotification } from './caNotification';
 import { DepOutputChannel } from './DepOutputChannel';
+import { record, shutDown, startUp } from './redhatTelemetry'
 
 let lspClient: LanguageClient;
 
 export let outputChannelDep: any;
 
 export function activate(context: vscode.ExtensionContext) {
+  // record extension startup 

```

Comment:
Remove redundant comments. Comments should be used only if your code logic is hard to understand from the used constructs.
Ditto

Reply:
removed